docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -486,20 +486,32 @@ For a time interval starting at ‘StartTime’ and continuing for ‘Duration
 For example, `timeSlots(toDateTime('2012-01-01 12:20:00'), 600) = [toDateTime('2012-01-01 12:00:00'), toDateTime('2012-01-01 12:30:00')]`.
 This is necessary for searching for pageviews in the corresponding session.
 
-## formatDateTime(Time, Format\[, Timezone\]) {#formatdatetime}
+## formatDateTime {#formatdatetime}
 
 Function formats a Time according given Format string. N.B.: Format is a constant expression, e.g. you can not have multiple formats for single result column.
 
-Supported modifiers for Format:
-(“Example” column shows formatting result for time `2018-01-02 22:33:44`)
+**Syntax**
+
+``` sql
+formatDateTime(Time, Format\[, Timezone\])
+```
+
+**Returned value(s)**
 
-| Modifier | Description                                             | Example    |
+Returnes time and date values according to the determined format.
+
+**Replacement fields**
+These replacement fields or placeholders define pattern for the result-string. “Example” column shows formatting result for time `2018-01-02 22:33:44`.
+
+| Placeholder | Description                                             | Example    |
 |----------|---------------------------------------------------------|------------|
 | %C       | year divided by 100 and truncated to integer (00-99)    | 20         |
 | %d       | day of the month, zero-padded (01-31)                   | 02         |
 | %D       | Short MM/DD/YY date, equivalent to %m/%d/%y             | 01/02/18   |
 | %e       | day of the month, space-padded ( 1-31)                  | 2          |
 | %F       | short YYYY-MM-DD date, equivalent to %Y-%m-%d           | 2018-01-02 |
+| %G       | four-digit year format for ISO week number, calculated from the week-based year defined by the ISO 8601 standard, normally useful only with %V  | 2018         |
```

Comment:
Let's place the https://en.wikipedia.org/wiki/ISO_8601#Week_dates link in this description. It can help users to quicker understand the difference between week numbering year and ordinary calendar year.